workspaces/global-header/app-config.yaml

```
@@ -32,6 +32,15 @@ backend:
     connection: ':memory:'
   # workingDirectory: /tmp # Use this to configure a working directory for the scaffolder, defaults to the OS temp-dir
 
+  # enable notifications backend
+  auth:
+    dangerouslyDisableDefaultAuthPolicy: true
+    externalAccess:
+      - type: static
+        options:
+          token: ${EXTERNAL_SERVICE_TOKEN}
```

I changed it locally to this and it just worked fine. Wdyt?

```suggestion
          token: EXTERNAL_SERVICE_TOKEN
```